addons/website/models/ir_attachment.py

```diff
@@ -9,4 +9,4 @@ class Attachment(models.Model):
     _inherit = "ir.attachment"
 
     # related for backward compatibility with saas-6
-    website_url = fields.Char(string="Attachment URL", related='local_url', deprecated=True)
+    website_url = fields.Char(string="Website URL", related='local_url', deprecated=True)
```

Comment:
are we sure this field "for backward compatibility with saas-6" cannot be removed ? :D